CONTRIBUTING.md

```diff
@@ -81,7 +81,11 @@ The next section contains more information on the workflow followed for Pull Req
 
 - The Maintainers will apply the proper labels for the issue. This may include an indication of the subject area or type (e.g. `deprecation`).
 
-**4.3.	Reviewing/Discussion.**
+**4.3.	Unit Test Policy**
+
+- Any change to the FDC3 specification typescript codebase must be supoorted by accompanying unit tests.
```

Comment:
Some more detail would also be useful here - FDC3 is made up of 4 specifications, one of which relates to the Desktop Agent API. Most of the codebase is just typing to define that API standard. However, there is actual code in method.ts, which is currently all that we apply unit tests to. There is also code in ContextTypes.ts, which is auto-generated from context schemas. This does not currently include tests - and we might not want to be forced to add them when new Context type schemas are added and start autogeneratingnew functions.

There are other apps in the /toolbox directory. These don't currently have unit tests.
We might add further apps there, e.g. a basic app directory implementation - the existing out-of-date implementation was recently removed.